### PR TITLE
Fix plotting docstrings hardcoding

### PR DESCRIPTION
**Pull request by psygo**
The two improvements of this *pull request* are:

1. More detailing to the `docstrings` and some minor improvements on formatting consistency
1. Fixes to some parameter *hard coding* on some plots (only visual stuff).

I've made a slight (*cough*) git mistake: this is building upon my [Beta-Geo Fitter Docstrings](https://github.com/CamDavidsonPilon/lifetimes/pull/290) other *pull request*, let me know if it's too big a problem and I will try to solve it.

### CODE REVIEW

**Comment by CamDavidsonPilon** [comment]
oops, I may have prematurely un-drafted this (drafts are not usually part of my workflow). Sorry!

**Comment by psygo** [comment]
No problem about the undrafting of this *PR*. I had only done so because I had made a mistake with Git.

But I don't quite understand what's going on here. Travis CI's checks failed but somehow it has no conflicts?

I've only altered docstrings and added some parameters, which are mostly very trivial to debug also. I will take a closer look when I have time again but if someone has a clue, please post it.

File: lifetimes/plotting.py [hunk 1]
```diff
@@ -662,16 +740,35 @@ def plot_dropout_rate_heterogeneity(
     fig, ax = plt.subplots(1)
     fig.suptitle(suptitle, fontsize=suptitle_fontsize, fontweight="bold")
 
-    ax.set_title("mean: {:.3f}, var: {:.3f}".format(beta_mean, beta_var))
+    ax.set_title("{}: {:.3f}, {}: {:.3f}".format(title_list[0], rate_mean, title_list[1], rate_var))
```
Comment (CamDavidsonPilon) [inline lifetimes/plotting.py]: here's the problem: `rate_mean` is not defined

Comment (psygo) [reply lifetimes/plotting.py]: Ah, yes, stupid copy paste problem. I'll fix it tomorrow.

**Comment by psygo** [comment]
@CamDavidsonPilon I thought you were on vacation or away for some reason, but now I see that you're busy developing another project. 

Sorry for being a bit annoying, but would it be possible if you gave me a hand with this Issue #271? It has been quite nightmarish, and it could cause me problems with my job I believe.

If you would rather chat elsewhere, my email is philippefanaro@gmail.com

File: lifetimes/plotting.py (hunk 1 above)
Comment (psygo) [reply lifetimes/plotting.py]: (Now it's fixed, I guess?)

**Pull request closed without merging**